tests/c/conformance_test.c

```diff
@@ -2869,6 +2864,9 @@ static int test_conn(int argc, char *argv[], SQLHANDLE hconn)
   }
 
   if (r == 0) {
+    r = test_conn_SQL_catalog_functions(hconn);
```

Comment:
> as what i recall, conn's catalog function does not require connected or not.

It turned out that inside the test_conn_SQL_catalog_functions, I would connect to the SQLSERVER_ODBC_DSN data source, because I was mainly testing the ODBC behavior of the TDengine, so I passed in the connection handle. If we feel that such behavior is not in line with our willingness, we can discuss and modify it

Reply:
I'll reconsider